Add a manual test script

hortongn opened the pull request:
Fixes #104 

Adds a manual test script we can follow before deploys to verify everything is working.

The script covers basic functionality and attempts to verify unique features and overrides we've added to the application.

If you come across incorrect directions, ambiguous directions, missing directions, or think that something that should be tested isn't being covered, please request changes.

chuckgreenman commented:
Hey @hortongn !  Made my way through the test script, here are some observations that I made:

## Section 1

1. We may want to specify that you only get welcome emails when the app is running on a server.

## Section 2

1. Should be (Lastname, Firstname)
1. If you have trouble uploading a word file, make sure you have libre office installed
	1. brew cask install libreoffice
1. Can't attach a file from the box app from localhost, err: invalid_client

## Section 3

## Section 4
1. Don't forget to create collection type: `bundle exec rails hyrax:default_collection_types:create`
1. Having trouble changing the visibility of a collection to private.

## Section 5
1. Shared student work won't show up on the 'My Works' tab. Find it on 'Browse' instead.

## Section 6
1. Search and Discovery sections mentions that the department facet will only show up if another facet is choosen, soon, it will only show up if a college facet is set.

## Section 7
1. Running jobs inline, scholar happily uploads, displays and allows for the download of the EICAR file.
1. Google CAPTCHA is shown, key invalid locally, works on scholar.uc.edu

## Section 8
1. ETD Type isn't showing up when logged in as admin@example.com.

hortongn commented:
@chuckgreenman I made some changes to my PR based on your feedback.  For the moment, my changes are in a second commit in this PR so that you can see what I changed.  After everything looks good I will squash the commits before you merge.

Add another comment if there are any additional changes you would like to see.

## Section 1

1. We may want to specify that you only get welcome emails when the app is running on a server.
**I've addressed this in the new commit**

## Section 2

1. Should be (Lastname, Firstname)
**I've addressed this in the new commit**
1. If you have trouble uploading a word file, make sure you have libre office installed
	1. brew cask install libreoffice
**I've addressed this in the new commit**
1. Can't attach a file from the box app from localhost, err: invalid_client
**I've addressed this in the new commit (mentioned that it only works from servers**

## Section 3

## Section 4
1. Don't forget to create collection type: `bundle exec rails hyrax:default_collection_types:create`
**Since this is in the README now, I think we can leave it out of the script**
1. Having trouble changing the visibility of a collection to private.
**This looks like it might be a legitimate bug.  I'll open an issue for it.  No changes are needed in this script**

## Section 5
1. Shared student work won't show up on the 'My Works' tab. Find it on 'Browse' instead.
**The work would have shown up under the "Managed Works" tab instead of your works**

## Section 6
1. Search and Discovery sections mentions that the department facet will only show up if another facet is choosen, soon, it will only show up if a college facet is set.
**I've addressed this in the new commit**

## Section 7
1. Running jobs inline, scholar happily uploads, displays and allows for the download of the EICAR file.
**I've addressed this in the new commit (mentioned that it only works on servers)**
1. Google CAPTCHA is shown, key invalid locally, works on scholar.uc.edu
**I've addressed this in the new commit (mentioned that it only works on servers)**

## Section 8
1. ETD Type isn't showing up when logged in as admin@example.com.
**Being an admin allows me to see the ETD work type.  Let's try it on your machine together**
